website/pages/api-docs/auth/userpass/index.mdx

```diff
@@ -27,7 +27,7 @@ Create a new user or update an existing user. This path honors the distinction b
 
 ### Parameters
 
-- `username` `(string: <required>)` – The username for the user.
+- `username` `(string: <required>)` – The username for the user. Accepted characters: alphanumeric plus "\_", "-", "." (underscore, hyphen and period).
```

Comment:
```suggestion
- `username` `(string: <required>)` – The username for the user. Accepted characters: alphanumeric plus "_", "-", "." (underscore, hyphen and period).
```

The hyphen and period cannot appear first.  This seems like an arbitrary restriction, but should we mention it to be more precise?